udata/tests/api/test_dataservices_api.py

```diff
@@ -15,6 +16,11 @@
 from . import APITestCase
 
 
+def dataservice_in_response(response: TestResponse, dataservice: Dataservice) -> bool:
+    only_dataservice = [r for r in response.json["data"] if r["id"] == str(dataservice.id)]
```

Comment:
I was thinking "meh, this name feels a bit off, i'm not sure what's the logic behind it", and then I saw it was my code ... guess I couldn't come up with a better one, and still can't 🙄